app/components/public/ticket-list.js

```diff
@@ -16,7 +16,7 @@ export default Component.extend(FormMixin, {
   }),
 
   shouldDisableOrderButton: computed('isUnverified', 'hasTicketsInOrder', function() {
-    return !this.hasTicketsInOrder;
+    return !this.hasTicketsInOrder || !this.donationPriceValid;
```

Comment:
@mrsaicharan1 add 'donationPriceValid' in the watched properties

Reply:
@uds5501 Tried it but no luck :/

Reply:
This should be `return !(this.hasTicketsInOrder && this.donationPriceValid);`